pandas/tests/groupby/aggregate/test_numba.py

```diff
@@ -15,7 +15,7 @@
 from pandas.core.util.numba_ import NUMBA_FUNC_CACHE
 
 
-@td.skip_if_no("numba", "0.46.0")
+@td.skip_if_no("numba")
```

Comment:
prob could remove these individual skips and just skip the entire module if numba not installed?

Reply:
True. All `*/test_numba.py` tests should require numba